sourcegraph/sourcegraph
# create migrations for file embeddings

Closes #54679

This PR creates the database tables needed to track file uploads for the worker to pick up and act on. 
When a file has been successfully uploaded to the `blob store`, we'll add a record for it to be picked up by the embedding worker in the `file_embedding_jobs` table.

## Test plan


* Run `sg migrate up`
 The database tables `file_embedding_jobs` and `file_embedding_jobs_stats` should be created in your `sourcegraph` database.

* Run `sg migrate undo -db=frontend`
This should revert the creation of the tables above.
